### DISCUSSION TITLE
Kuka - interpolation?

### DISCUSSION THREAD

**Discussion by toniosterlund** [visose/Robots#152] [Q&A] [answered]
Hi @visose ,
how would one go about adding C_DIS interpolation to the toolpath using Kuka and LIN targets?

Also, are the 'zones' explained somewhere? Kinda feel that it's related...

**Comment by visose** [comment, marked as answer]
Hi, yes, use the Zone parameter to set C_DIS in KUKA. There should be an explanation in the Parameters section of the wiki.
